src/lib/dapp/index.ts

```diff
@@ -429,11 +350,43 @@ export class DAppConnector {
    * const result = await dAppConnector.executeTransaction(params)
    * ```
    */
-  public async executeTransaction(params: ExecuteTransactionParams) {
-    return await this.request<ExecuteTransactionRequest, ExecuteTransactionResult>({
-      method: HederaJsonRpcMethod.ExecuteTransaction,
-      params,
-    })
+  public async executeTransaction(
+    params: ExecuteTransactionParams,
+  ): Promise<ExecuteTransactionResult['result']>
+
+  /**
+   * Executes a transaction on the Hedera networkw with a signer
+   *
+   * @param {AccountId} accountId - AccountId of the signer
+   * @param {Transaction} transaction - Transaction to be executed
+   * @returns {Promise<Transaction>}
+   *  * @example
+   * ```ts
+   * const params = {
+   *  signedTransaction: [transactionToBase64String(transaction)]
+   * }
+   *
+   * const result = await dAppConnector.executeTransaction(AccountId.fromString('0.0.12345), transaction)
+   * ```
+   */
+  public async executeTransaction(
```

Comment:
I'm questioning if we should remove the execute*/sign* methods from the DAppConnector entirely and instead only use the Signer for executing transactions.

It feels like the DAppConnector should limit its responsibility managing walletconnect sessions and extending walletconnect.

Reply:
I agree with that but I keep there to be backwards compatible with the previous versions.